src/specs/components/page-header/demo/data--background-image.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  title: 'Page title',
+  description:
```

Comment:
instead of having multiple data files we could have just one containing everything and handle it in the stories to display some elements or not.
that's how we handled latest component

Reply:
done